react-common/styles/onboarding/TeachingBubble.less

```diff
@@ -32,6 +32,16 @@
     border-radius: 0.5rem;
     padding: 1rem;
     z-index: @modalDimmerZIndex;
+
+    .common-button {
+
+        &:focus {
+            outline: .1rem solid @white;
+        }
+        &:focus::after {
```

Comment:
do we normally set outline using an ::after pseudoelement that this is overriding?

Reply:
Found it. I think it's this: https://github.com/microsoft/pxt/blob/master/react-common/styles/controls/Button.less#L158